Makefile

```diff
@@ -77,56 +79,52 @@ bin/zsdcc$(EXESUFFIX):
 	cd $(SDCC_PATH) && mv ./bin/sdcpp $(Z88DK_PATH)/bin/zsdcpp
 	$(RM) -fR $(SDCC_PATH)
 
-bin/appmake$(EXESUFFIX):
+bin/appmake$(EXESUFFIX): $(wildcard src/appmake/*.c src/appmake/*.h)
```

Comment:
Maybe we should go back to .PHONY targets here? I'm not too keen on this top-level makefile having incomplete dependencies for the binaries.

By incomplete I mean they don't reference the modules that are used.

Reply:
Agree with .PHONY. I'll revise and commit.

Reply:
Thanks Paulo, sorry I was at fault changing it initially